Sources/BraintreePayPalMessaging/BTPayPalMessagingOfferType.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import PayPalMessages
+
+/// Preferred message offer to display
+/// - Note: This module is in beta. It's public API may change or be removed in future releases.
+public enum BTPayPalMessagingOfferType {
+
+   /// Pay Later short term installment
+   case payLaterShortTerm
+
+   /// Pay Later long term installments
+   case payLaterLongTerm
+
+   /// Pay Later deferred payment
+   case payLaterPayInOne
+
+   /// PayPal Credit No Interest
+   case payPalCreditNoInterest
+
+   var offerTypeRawValue: PayPalMessageOfferType {
+       switch self {
+       case .payLaterShortTerm:
+           return .payLaterShortTerm
+       case .payLaterLongTerm:
+           return .payLaterLongTerm
+       case .payLaterPayInOne:
+           return .payLaterPayIn1
```

Comment:
does this need to be changed to `payLaterPayIn1`?

Reply:
Intentionally changed our naming to `one` based on this discussion: https://github.com/braintree/braintree_ios/pull/1153#discussion_r1424484439. But this is returning the raw type in the messaging SDK which uses `1` so they will differ.

Reply:
awww I see. Thought it was a hard limitation, thanks for confirming!